docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md

```diff
@@ -101,16 +99,9 @@ cat <<EOF | kubectl apply -f -
      namespace: default
    spec:
      sink: 'http://lastorder.default.svc.cluster.local'
-     filter:
-       filters:
-         - eventSource:
-             property: source
-             type: exact
-             value: ''
-           eventType:
-             property: type
-             type: exact
-             value: sap.kyma.custom.myapp.order.payment-success.v1
+     source: myapp
```

Comment:
don't forget to change the `apiVersion` to `eventing.kyma-project.io/v1alpha2`.

Reply:
Updated.